src/.nuget/toolchain.ubuntu.14.04-arm64.Microsoft.DotNet.ILCompiler.Development.nuspec

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0"?>
+<package >
+  <metadata>
+    <id>toolchain.ubuntu.14.04-arm64.Microsoft.DotNet.ILCompiler.Development</id>
```

Comment:
Here as written means the package has binaries that run natively on arm64. To cross compile, the name should be
toolchain.ubuntu.14.04-x64.Microsoft.DotNet.ILCompiler.Arm64. If we will eventually have a single target agnostic ILC, then Arm64 suffix is not needed in the package name.